torch/_torch_docs.py

```diff
@@ -3578,6 +3578,11 @@ def merge_dicts(*dicts):
 
 The output tensor is 1-D of size :attr:`steps`.
 
+.. warning::
+    Not providing a value for :attr:`steps` is deprecated, and in a future release
```

Comment:
it's not obvious what this means.  The default is 100 -- does that mean not providing an value?  Do you want to leave `steps=100` above for clarity or do you want to change it to None?

Reply:
That's a good point and I agree we can be clearer here. Why don't we do this:

- remove the default value in the doc string
- update the arg note to say that failing to provide the value will result in steps=100 but is deprecated
- updating the warning note to say that not providing a value will result in the deprecated behavior of steps=100

Reply:
Well None is probably bad given it sounds like there would be no steps...

Reply:
Sorry, how do you mean? I'm not suggesting we use None. Basically the docstring will look like the future one but the docs will say the deprecated behavior is allowed.

Reply:
Offline review: document as if it's the new version (doc test may complain) or keep it None and document that we hate that